assets/scripts/events.js

```diff
@@ -116,6 +122,9 @@ const addToCart = function (event) {
 }
 
 const onCheckout = () => {
+  // Hide update and delete buttons when cart is ready for purchase
```

Comment:
i don't agree with this. Checkout is creating an empty array, so it doesn't matter at that point if someone adds or removes the item. When someone updates the quantity, the update button recalcs the amount and sets the qty, and then 'purchase' updates the cart array with that information.